src/app/components/Search/Instances.jsx

```diff
@@ -1,66 +1,50 @@
 import React from 'react';
```

Comment:
Is "Instances" related to elasticsearch?

Reply:
Instances relates to the data structure language used for describing items which would be the equivalent in Sierra.